autoload/SpaceVim/layers/lang/nix.vim

```diff
@@ -0,0 +1,24 @@
+"=============================================================================
+" nix.vim --- nix language support for SpaceVim
+" Copyright (c) 2018-2019 Ben Gamari
```

Comment:
this line should be `" Copyright (c) 2016-2019 Wang Shidong & Contributors`

Reply:
Fixed.